.gitignore

```diff
@@ -0,0 +1,5 @@
+
+/target
+**/*.rs.bk
+Cargo.lock
+.idea
```

Comment:
May want to add 
```
.floo*
/rls
```
(We might want to come up with a common skeleton of these, come to think of it.)

Reply:
Global .gitignore is ideal for these, as we should only be excluding things the build system or language drop in there that shouldn't be there. IDE specific things that shouldn't be included are user-specific, and should be excluded using global .gitignore (such as that .idea dir).